### ISSUE TITLE
Android app shows "Markets" product list as "Options, Spot, Perpetuals, Futures" while IOS app has  "Options, Perpetuals, Futures, Spot"

### ISSUE THREAD

**Issue by laisee** [Power-Trade/api-docs#26]
A minor thing but inconsistent across platforms. Checked the Web UI and it only shows "Options, Perpetuals, Spot" with no "Futures" displayed.

Android app is version 2.2.1
IOS app is version 3.0.8

**Comment by laisee** [comment]
Re-tested on 04/Sep/24 and same issue observed - inconsistent sequence of product types between IOS and Android apps

**Comment by trungpwt** [comment]
@laisee yes, I'm doing improvements and bug fixes for ADR, as you can see that is already updated on iOS. Please wait til I complete it. Thanks.

**Comment by laisee** [comment]
Ok, no problem pls let me know when there are any updates to teat or re-test
